components/logins/src/login.rs

```diff
@@ -342,6 +342,19 @@ pub struct RecordFields {
     pub times_used: i64,
 }
 
+/// A login together with record fields, handed over to the store API; ie a login persisted
+/// elsewhere, useful for migrations
+pub struct LoginEntryWithRecordFields {
+    pub entry: LoginEntry,
+    pub record: RecordFields,
```

Comment:
"RecordFields" is a pre-existing term, but I've never liked it and now it's becoming part of the public API.  If you have a better name, I'd be happy to switch to that.  If not, how about renaming it to `LoginRecordFields`.  That avoids some issues on Swift which doesn't have any modules / name spaces.